xml/System.Security.AccessControl/ObjectSecurity.xml

```diff
@@ -1306,7 +1306,7 @@
         <param name="preserveInheritance">
           <see langword="true" /> to preserve inherited access rules; <see langword="false" /> to remove inherited access rules. This parameter is ignored if <paramref name="isProtected" /> is <see langword="false" />.</param>
         <summary>Sets or removes protection of the access rules associated with this <see cref="T:System.Security.AccessControl.ObjectSecurity" /> object. Protected access rules cannot be modified by parent objects through inheritance.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>When you call the method with isProtected=true and preserveInheritance=true, you need to walk the new ACL of the object and check for DENY type ACEs. For a canonically sorted DACL, the DENY ACEs must appear in the front of the DACL.</remarks>
```

Comment:
Tiny suggestion so the parameters show up nicely formatted:

```suggestion
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks
When you call the method with `isProtected=true` and `preserveInheritance=true`, you need to walk the new ACL of the object and check for DENY type ACEs. For a canonically sorted DACL, the DENY ACEs must appear in the front of the DACL.
 
        ]]></format>
      </remarks>
```

Reply:
@HerbertMauerer do you have a chance to apply the suggestion? That way we can get this merged.